Grammar fixes (#1966)
hacks-guide/Guide_3DS: _pages/en_US/ctrtransfer.txt

  + This prompt will only appear if you have an EmuNAND
1. Press (A) to unlock SysNAND (lvl1) writing, then input the key combo given
  + This process will take some time

> 1. Once the transfer has been completed, press (A) to continue

Reviewer: ```suggestion
1. Once the transfer is complete, press (A) to continue
```

Reviewer: This was not added.
